service/frontend/workflow_handler.go

```diff
@@ -2817,6 +2821,62 @@ func (wh *WorkflowHandler) DescribeSchedule(ctx context.Context, request *workfl
 	}, nil
 }
 
+func (wh *WorkflowHandler) annotateSearchAttributesOfScheduledWorkflow(
+	queryResponse *schedspb.DescribeResponse,
+	nsName string,
+) error {
+	ei, err := wh.getScheduledWorkflowExecutionInfoFrom(queryResponse)
+	if err != nil {
+		return err
+	}
+	annotatedAttributes, err := wh.createAnnotatedSearchAttributesFromNewWorkflow(ei, nsName)
+	if err != nil {
+		return fmt.Errorf("annotate search attributes:%w", err)
```

Comment:
I don't think we need to wrap these errors here

Reply:
This tells the error receiver the whole path. It is why we do not neet to tell that it is annotation of attribute failed in the caller code.